packages/client/src/generation/TSClient/jsdoc.ts

```diff
@@ -326,4 +326,37 @@ const { count } = await ${ctx.method}({
       where: (singular, plural) => `Filter which ${plural} to delete`,
     },
   },
+  aggregateRaw: {
+    body: (ctx) =>
+      `Perform aggregation operations on a ${ctx.singular}.
+@param {${getModelArgName(ctx.model.name, ctx.action)}} args - Select which aggregations you would like to apply.
+@example
+const ${lowerCase(ctx.mapping.model)} = await ${ctx.method}({
+  pipeline: [
+    { $match: { status: "registered" } },
+    { $group: { _id: "$country", total: { $sum: 1 } } }
+  ]
+})`,
+    fields: {
+      pipeline: () =>
+        'A sequence of data aggregation operations or stages ${@link https://docs.mongodb.com/manual/reference/command/aggregate/#command-fields MongoDB Docs}.',
+      options: () =>
+        'Additional options that passes to the aggregate command ${@link https://docs.mongodb.com/manual/reference/command/aggregate/#command-fields MongoDB Docs}.',
+    },
+  },
+  findRaw: {
+    body: (ctx) =>
+      `Find zero or more ${ctx.plural} that matches the filter.
```

Comment:
```suggestion
      `Find zero or more ${ctx.plural} that matches the filter.
```
Plural doesn't work on all names, zero or more is probably enough to convey the meaning here with singular?

Reply:
Ha wait, there's a lot of them :sweat_smile: I'll open an issue instead.

Reply:
https://github.com/prisma/prisma/issues/11434